pkgs/stdenv/linux/default.nix

```diff
@@ -468,15 +511,7 @@ in
   # binutils built.
   #
   # resulting stage5 (final) stdenv:
-  # - coreutils, binutils: from nixpkgs, built by nixpkgs toolchain
-  # - glibc: from nixpkgs, built by bootstrapFiles toolchain
-  # - gcc: from nixpkgs, built by bootstrapFiles toolchain. Can assume
-  #        it has almost no code from bootstrapTools as gcc bootstraps
-  #        internally. The only exceptions are crt files from glibc
-  #        built by bootstrapTools used to link executables and libraries,
-  #        and the bootstrapTools-built, statically-linked
-  #        lib{mpfr,mpc,gmp,isl}.a which are linked into the final gcc
-  #        (see commit cfde88976ba4cddd01b1bb28b40afd12ea93a11d).
+  # - coreutils, binutils, glibc, gcc: from nixpkgs, built by bootstrapFiles toolchain
```

Comment:
Should be "built by nixpkgs toolchain" probably?

Reply:
Oh, good catch! Pushed the update.